LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/AltIdSource.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.labapi.utilities.constants;
+
+public enum AltIdSource {
+    NONE(LabConstants.AltIdSource.NONE),
+    ON_PREM(LabConstants.AltIdSource.ON_PREM);
+
+    final String value;
+
+    AltIdSource(final String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
```

Comment:
>getValue [](http://example.com/codeflow?start=18&length=8)

Use toString() instead of getValue, That way you can use it as AltSource.None instead of AltSource.None.getValue(), and will also not need to do a nul check.

Same for all other enums

Reply:
I tried doing it but it doesn't seem to work...see comment here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1406#discussion_r648628660

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1406/commits/d5e4b3943a0944faaab79d0fecb91d175da48c72